sys-process/btop/files/btop-1.2.13-allow-clang.patch

```diff
@@ -0,0 +1,234 @@
+Bug: https://bugs.gentoo.org/839318
+Upstream Commit: https://github.com/aristocratos/btop/commit/7e50b03e391a346b2b6ce92be332c58d91f75a000w
+
+From af7d5086fce8d995f3f8b10677d107a1124a12bf Mon Sep 17 00:00:00 2001
+From: "Jakob P. Liljenberg" <admin@qvantnet.com>
+Date: Thu, 15 Jun 2023 17:45:05 +0200
+Subject: [PATCH] Merge pull request #510 from nobounce/LLVM
```

Comment:
This info doesn't really need to be in the patch file in the tree, does it?

Reply:
Probably not. scrub-patch didn't remove it so I didn't realize it was still there.

Reply:
Which part? This looks fine.

Reply:
I don't think this change was right. Having the metadata is important. You can keep the headers and just say "(with hunks from)" or something if you're modifying it.